Sources/TSCUtility/dlopen.swift

```diff
@@ -62,8 +62,10 @@ public struct DLOpenFlags: RawRepresentable, OptionSet {
     public static let deepBind: DLOpenFlags = DLOpenFlags(rawValue: 0)
   #else
     public static let first: DLOpenFlags = DLOpenFlags(rawValue: 0)
-  #if !os(Android)
+  #if os(Linux)
     public static let deepBind: DLOpenFlags = DLOpenFlags(rawValue: RTLD_DEEPBIND)
+  #else
+    public static let deepBind: DLOpenFlags = DLOpenFlags(rawValue: 0)
```

Comment:
Before this change, this would not be defined on Android, but with this change it will. Should this be `#else if os(OpenBSD)` instead?

Reply:
On the one hand, the code pattern nearby suggests that undefined flags are set to 0, but the actual symbol is not defined conditionally per OS as you point out. 

This is easy enough to adjust but -- because this was a negative check (`!os(Android)`) this means that we always _assume_ this symbol exists by default and requires porters to make affirmative exclusions for new platforms.

What would you prefer?

Reply:
@drexin wdyt?

Reply:
Since it was not defined on Android before, I would prefer it to stay that way. I think adding an empty `#else if os(Android)` should be okay.

Reply:
Good idea. Done.

Reply:
I added the Android check because `RTLD_DEEPBIND` is not defined in Bionic. I just tried building for Android with this last line you added instead, and this works too. Please remove the Android check altogether here and [this other dependent one](https://github.com/apple/swift-package-manager/commit/946f29ae260d6781666e5ad26697a77528969fc9#diff-de74314b412b8d423fec461826c4ad64056d9d77e95c65ace4e27eb969beb8acR196), would be good to clean this up for Android too.